playbooks/ansible-test-base/run.yaml

```diff
@@ -13,6 +13,15 @@
             file: '{{ _fetch.dest }}'
             name: galaxy_info
 
+    - name: Enable FIPS mode
+      when:
+        - ansible_distribution == "CentOS"
+        - ansible_test_network_cli_ssh_type == "libssh"
+        - hostvars[item].ansible_network_os in ['eos']
```

Comment:
Rather then network_os check, I wonder if we should add a new zuul var to the base appliance job call fipsmode: True, then jobs will disable it if they don't want it use it.

Thoughts?

Reply:
Sounds reasonable. I will add it.